features/step_definitions/stepdefs.js

```diff
@@ -700,12 +701,21 @@ When(
 );
 
 When(
-    /^an authenticated (\"(post|put)\"\s)?request is made to "(.*)" with a file$/,
+    /^an authenticated (\"(post|put)\"\s)?request is made to "(.*)" with an image$/,
     async function (method, path) {
-        // Create a test file
-        const file = new File(['test image content'], 'test.jpg', {
-            type: 'image/jpeg',
-        });
+        const buffer = await sharp({
+            create: {
+                width: 200,
+                height: 200,
+                channels: 3,
+                background: { r: 255, g: 0, b: 0 },
+            },
+        })
+            .toFormat('jpg')
+            .toBuffer();
```

Comment:
Seeming though this is an e2e test, we could just read a real file from the filesystem instead of programatically creating it? (see `fixtures` in other e2e tests). Maybe not worth the hassle? (just means it we ever decide to not use `sharp` we don't have to rip it out here)

Reply:
Yeah good idea! Feels better to work with a real image in a e2e test anyways. 

Fixed in c31dd1ccd9f436532e45e6ee41d627297bc323c9